orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -135,3 +152,32 @@ def _replicate_for_data_parallel(self):
 
         raise NotImplementedError("ORTModule is not compatible with torch.nn.DataParallel. "
                                   "Please use torch.nn.parallel.DistributedDataParallel instead.")
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                                missing_keys, unexpected_keys, error_msgs):
+        """Override original method to delegate execution to the base module"""
+
+        self.original_module._load_from_state_dict(state_dict, prefix, local_metadata, strict,
+                                missing_keys, unexpected_keys, error_msgs)
+
+    def named_children(self):
+        """Override original method to delegate execution to the base module"""
+
+        yield from self.original_module.named_children()
+
+    @property
+    def original_module(self):
+        """Accessor for the original PyTorch module
+
+        Users can retrieve the original module by using this property
+        ort_model = ORTModule(model)
+        original_model = ort_model.original_module
+        """
+
+        return self._module.original_module()
+
+    @property
+    def _flattened_module(self):
+        """Accessor for the flattened module"""
+
+        return self._module.flattened_module()
```

Comment:
This is only need for the tests, right? For such, we could directly access internal module until the the task is resolve (either by exposing the original module, by exposing method registration APIs or by automatically adding them to ORTModule API (kind of dangerous)

The main point is not mix fixes in a single PR